web-workers/fibonacci-worker/LICENSE

```diff
@@ -0,0 +1,116 @@
+CC0 1.0 Universal
```

Comment:
Same here, it will be covered by the license here: https://github.com/mdn/dom-examples/blob/master/LICENSE